src/pages/patientView/PatientViewPage.tsx

```diff
@@ -193,6 +199,18 @@ export default class PatientViewPage extends React.Component<IPatientViewPagePro
         showCustomTab(div, tab, this.props.routing.location, patientViewPageStore);
     }
 
+    private wholeSlideViewerUrl = remoteData({
+        await: () => [patientViewPageStore.getWholeSlideViewerIds],
+        invoke: async() => {
+            const url = getWholeSlideViewerUrl(patientViewPageStore.getWholeSlideViewerIds.result!, this.props.appStore.userName!);
+            await request.get(url);
```

Comment:
you should probably put a comment to explain this logic.   i.e. if request succeeds then we return the url because we know request works.   but if request fails, we will return undefined.    
in fact, a more clear way of doing this would be simple to wrap request in try {} catch {}  rather than use the onError handler, which slightly confuses things.

Reply:
Thanks for your help, @alisman. I just updated this pr. It is ready to review now.